butane_core/src/migrations/fsmigrations.rs

```diff
@@ -161,7 +161,7 @@ impl MigrationMut for FsMigration {
             TYPES_FILENAME,
             serde_json::to_string(&types)
                 .map_err(|e| {
-                    eprintln!("failed to read types");
+                    eprintln!("failed to write types");
```

Comment:
alternatively, this line could be removed, but I think it is helpful until we switch to using `log` throughout.

e.g. the eprintln above (the real "read" error) appears at https://github.com/Electron100/butane/issues/162

Also potentially this should be
```rust
eprintln!("failed to write types {typefile:?}");
```
similar to the read error above.